tests/src/Util/DateTimeFormatTest.php

```diff
@@ -87,4 +87,58 @@ public function testApiDate()
 	{
 		self::assertEquals('Wed Oct 10 00:00:00 +0000 1990', DateTimeFormat::utc('1990-10-10', DateTimeFormat::API));
 	}
+
+	public function dataFix(): array
+	{
+		return [
+			'Mo, 19 Sep 2022 14:51:00 +0200' => [
```

Comment:
Does this value really have to be duplicated in both array key and the "dateString" array parameter?

Reply:
Yes, the key of the data array is used in the error message when there's a failure. It's more explicit than "test 0 has failed".